.changeset/thirty-moles-invite.md

```diff
@@ -0,0 +1,13 @@
+---
+"@comet/admin-babel-preset": minor
+"@comet/admin-color-picker": minor
+"@comet/admin-date-time": minor
+"@comet/admin-icons": minor
+"@comet/admin-rte": minor
+"@comet/admin-theme": minor
+"@comet/admin": minor
+"@comet/blocks-admin": minor
+"@comet/cms-admin": minor
+---
+
+Add support for React 18
```

Comment:
Shall we probably mention React 18 upgrade in the migration guide and mention the existing codemods for react 18? https://github.com/eps1lon/types-react-codemod - for example the [implicit-children](https://github.com/eps1lon/types-react-codemod?tab=readme-ov-file#implicit-children-react-18) codemod?